### ISSUE TITLE
Airlocks and firelocks that were blown up can't be anchored

### ISSUE THREAD

**Issue by Alekshhh** [space-wizards/space-station-14#14054]
## Description
As title explains, if a bomb unanchors a firelock or airlock, they can't be wrenched anymore and can't be deconstructed*
*not tested on deconstruction but I believe I tried to do it on a firelock and failed because it couldn't be anchored

**Reproduction**
Explosion near airlock or firelock, try to anchor it, fail, cry

**Comment by Emisse** [comment]
this only happens from gravity anomaly not explosions

**Comment by Alekshhh** [comment]
ah that makes sense, I'll tweak the wording

**Comment by lzk228** [comment]
yeah they cannot be deconstructed because deconstructing needs anchoring
only way to get rid of them is rcd

**Comment by SlamBamActionman** [comment]
You can anchor these now.
